resources/models/resource.py

```diff
@@ -378,3 +392,33 @@ class Meta:
         verbose_name = _('resource image')
         verbose_name_plural = _('resource images')
         unique_together = (('resource', 'sort_order'),)
+
+
+class ResourceEquipment(ModifiableModel):
+    resource = models.ForeignKey(Resource, related_name='resource_equipments')
+    equipment = models.ForeignKey(Equipment, related_name='resource_equipments')
+    data = hstore.DictionaryField(null=True, blank=True)
+    description = models.TextField(blank=True)
+
+    objects = hstore.HStoreGeoManager()
+
+    class Meta:
+        verbose_name = _('resource equipment')
+        verbose_name_plural = _('resource equipments')
+
+    def __str__(self):
+        return "%s / %s" % (self.equipment, self.resource)
+
+
+class EquipmentAlias(ModifiableModel, AutoIdentifiedModel):
```

Comment:
Would love to have a docstring for this class (the usage is not directly clear).